### ISSUE TITLE
NVDA + Chrome: Issue with Keyboard Nav of Soccer Balls

### ISSUE THREAD

**Issue by kathy-phet** [phetsims/mean-share-and-balance#355]
I was thinking about sharing MSaB with a blind user to explore, and was taking it for a spin with NVDA. 

I ran across a keyboard nav issue. After I had put a bunch of soccer balls out onto the field and was hoping to grab and move one of them. NVDA somehow was blocking the "space" bar command. I could not "grab" a ball when I was in NVDA. (It also did not speak "press space to grab"). So I could not do anything here. 

@Nancy-Salpepi or @KatieWoe - Can you just see if you can reproduce this issue? If so, it might be a common code issue with the grab sort code and NVDA.

![image](https://github.com/user-attachments/assets/3cc2bef0-f69d-4a80-ad32-801b966cfe37)

**Comment by KatieWoe** [comment]
I didn't have trouble picking up the soccer balls, though it didn't say anything about space bar to pick up. I also noticed that the page on the website lists voicing and camera input, which the sim doesn't have, and mobile screen reader support, when no screen reader support was listed as supported when QA was testing the RC: https://github.com/phetsims/qa/issues/1138

**Comment by KatieWoe** [comment]
The issue of incorrect features seems general, and only on the sim page, not on the filter page.

**Comment by Nancy-Salpepi** [comment]
With NVDA when the group of soccer balls had the pink highlight, I first had to press the NVDA key + space tab to enter the group and then could use the arrow keys and space bar like usual. I also had to press the NVDA + space bar to exit and then switched back to tab. 
I also didn't hear anything about pressing space, but I wouldn't expect to hear help text because the sim doesn't currently have description. 

@kathy-phet anything else you would like QA to do here?

**Comment by Nancy-Salpepi** [comment]
Just an FYI--I looked at the a11y view for MSaB and there is no help text associated with grabbing the soccer balls:
<img width="1297" alt="Screenshot 2024-10-09 at 4 45 19 PM" src="https://github.com/user-attachments/assets/f0ab2aa2-9aa1-417f-ae53-ec020609ff3e">
